### PR TITLE
Updates glibc for CVE-2015-7547 and restarts services

### PR DESCRIPTION
**Pull request by nealbrown**

### CODE REVIEW

File: roles/security_errata/tasks/CVE-2015-7547.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  tags: CVE-2015-7547
+
+- name: kill neutron-ns-metadata-proxy
+  shell: pkill -f /usr/local/bin/neutron-ns-metadata-proxy
+  ignore_errors: true
+  when: glibc_minor_ver.stdout|int < 7
+  tags: CVE-2015-7547
+
+- name: restart neutron-dhcp-agent
+  service: name=neutron-dhcp-agent state=restarted
```
Comment (j2sol) [inline roles/security_errata/tasks/CVE-2015-7547.yml]: neutron-l3-agent

**Comment by nealbrown** [comment]
I do not believe that will work.  I believe it is started by neutron-dhcp-agent and this was confirmed by testing after craig's suggestion.  I'll test it again.

**Comment by nealbrown** [comment]
vagrant@allinone:~$ sudo pkill -f neutron-ns-metadata-proxy

vagrant@allinone:~$ ps aux|grep neutron-ns-metadata-proxy
vagrant 4119 0.0 0.0 11748 2204 pts/3 S+ 23:48 0:00 grep --color=auto neutron-ns-metadata-proxy
vagrant@allinone:~$ ps aux|grep neutron-ns-metadata-proxy
vagrant 4121 0.0 0.0 11748 2224 pts/3 S+ 23:48 0:00 grep --color=auto neutron-ns-metadata-proxy
vagrant@allinone:~$ ps aux|grep neutron-ns-metadata-proxy
vagrant 4123 0.0 0.0 11748 2328 pts/3 S+ 23:48 0:00 grep --color=auto neutron-ns-metadata-proxy
vagrant@allinone:~$ ps aux|grep neutron-ns-metadata-proxy
vagrant 4125 0.0 0.0 11748 2240 pts/3 S+ 23:48 0:00 grep --color=auto neutron-ns-metadata-proxy

vagrant@allinone:~$ sudo service neutron-l3-agent restart
neutron-l3-agent stop/waiting
neutron-l3-agent start/running, process 4141

vagrant@allinone:~$ ps aux|grep neutron-ns-metadata-proxy
vagrant 4149 0.0 0.0 11748 2224 pts/3 S+ 23:48 0:00 grep --color=auto neutron-ns-metadata-proxy
vagrant@allinone:~$ ps aux|grep neutron-ns-metadata-proxy
vagrant 4157 0.0 0.0 11748 2204 pts/3 R+ 23:48 0:00 grep --color=auto neutron-ns-metadata-proxy
vagrant@allinone:~$ ps aux|grep neutron-ns-metadata-proxy
vagrant 4161 0.0 0.0 11748 2232 pts/3 S+ 23:48 0:00 grep --color=auto neutron-ns-metadata-proxy
vagrant@allinone:~$ ps aux|grep neutron-ns-metadata-proxy
vagrant 4163 0.0 0.0 11748 2160 pts/3 S+ 23:48 0:00 grep --color=auto neutron-ns-metadata-proxy

vagrant@allinone:~$ sudo service neutron-dhcp-agent restart
neutron-dhcp-agent stop/waiting
neutron-dhcp-agent start/running, process 4197
vagrant@allinone:~$ ps aux|grep neutron-ns-metadata-proxy
neutron 4170 0.0 0.5 99568 35728 ? S 23:48 0:00 /opt/bbc/openstack-11.0-master/neutron/bin/python /usr/local/bin/neutron-ns-metadata-proxy --pid_file=/var/lib/neutron/external/pids/696623a5-47c9-400d-82ff-1b4bfff420d7.pid --metadata_proxy_socket=/var/lib/neutron/metadata_proxy --network_id=696623a5-47c9-400d-82ff-1b4bfff420d7 --state_path=/var/lib/neutron --metadata_port=80 --metadata_proxy_user=992 --metadata_proxy_group=992 --verbose --log-file=neutron-ns-metadata-proxy-696623a5-47c9-400d-82ff-1b4bfff420d7.log --log-dir=/var/log/neutron

**Comment by nealbrown** [comment]
retest

**Pull request merged**